### PR TITLE
Fix for REGISTRY-3233

### PR DESCRIPTION
**Pull request by Malintha**

### CODE REVIEW

File: modules/es-extensions/store/extensions/app/greg_store/pages/greg-landing.jag [hunk 1]
```diff
@@ -22,7 +22,9 @@ require('/modules/store.js').exec(function(ctx) {
 
 	var log = new Log();
 	log.debug("Printing user ############" + stringify(user));
-	var tenantId = user.tenantId;
+    
+    var tenantDetails = tenantApi.createTenantDetails(request, session);
```
Comment (splinter) [inline modules/es-extensions/store/extensions/app/greg_store/pages/greg-landing.jag]: Possible formatting issue

Comment (Malintha) [inline modules/es-extensions/store/extensions/app/greg_store/pages/greg-landing.jag]: I have actually formatted that before commit and cannot see this alignment issue on the IDE. Somehow Github shows it wrongly.

File: modules/es-extensions/store/extensions/app/greg_store/pages/greg-landing.jag [hunk 2]
```diff
@@ -22,7 +22,9 @@ require('/modules/store.js').exec(function(ctx) {
 
 	var log = new Log();
 	log.debug("Printing user ############" + stringify(user));
```
Comment (thushara35) [inline modules/es-extensions/store/extensions/app/greg_store/pages/greg-landing.jag]: Please initialise the log object and wrap the log with isDebugEnabled if condition.

Comment (Malintha) [inline modules/es-extensions/store/extensions/app/greg_store/pages/greg-landing.jag]: Fixed in https://github.com/wso2/product-greg/pull/856

File: modules/es-extensions/store/extensions/app/greg_store/pages/greg-landing.jag (hunk 1 above)
Comment (Malintha) [inline modules/es-extensions/store/extensions/app/greg_store/pages/greg-landing.jag]: selection_099

Comment (Malintha) [inline modules/es-extensions/store/extensions/app/greg_store/pages/greg-landing.jag]: selection_100

Comment (Malintha) [inline modules/es-extensions/store/extensions/app/greg_store/pages/greg-landing.jag]: Fixed in https://github.com/wso2/product-greg/pull/856/commits/9bf8e288c7bfad60a83fff703e0368b94d78f5ae

File: modules/es-extensions/store/extensions/app/greg-diff/pages/diff-view.jag [hunk 3]
```diff
@@ -5,15 +5,16 @@ require('/modules/store.js').exec(function (ctx) {
...
+    var tenantId = tenantDetails.tenantId;
     var options = ref.getQueryOptions(ctx.request.getQueryString());
     if (!options.type) {
         ctx.response.sendError(404, 'Unable to locate the asset without a valid Type');
         return;
     }
     var tenantResources = tenantApi.createTenantAwareAppResources(ctx.session);
     var appManager = tenantResources.appManager;
-    var page = ui.buildPage(ctx.session,ctx.request);
+    var page = ui.buildPage(ctx.session,ctx.request, tenantId);
```
Comment (sinthuja) [inline modules/es-extensions/store/extensions/app/greg-diff/pages/diff-view.jag]: Formatting issue. There should be a space between the two variables.

Comment (Malintha) [inline modules/es-extensions/store/extensions/app/greg-diff/pages/diff-view.jag]: @sinthuja Please find the re-factored code at b215dcac00050c70cac7fe85b1c57c99295c5b97
Thanks
Malintha

**Pull request merged**